homeassistant/components/vicare/strings.json

```diff
@@ -242,6 +242,44 @@
       "volumetric_flow": {
         "name": "Volumetric flow"
       },
+      "ess_state_of_charge": {
+        "name": "Battery"
+      },
```

Comment:
You could remove this one and the translation key for the device class translations

Reply:
```suggestion
```